src/odemis/gui/cont/project.py

```diff
@@ -134,8 +133,8 @@ def __init__(self, name, colour, tab_data, project_list, viewport):
         self._tab_data = tab_data
         self._project_bar = project_list
         self._viewport = viewport
-        self.regions_calib_2 = getattr(tab_data, "regions_" + "calib_2")  # FIXME calib_prefix
-        self.regions_calib_3 = getattr(tab_data, "regions_" + "calib_3")  # FIXME calib_prefix
+        self.regions_calib_2 = getattr(tab_data, "regions_calib_2")
+        self.regions_calib_3 = getattr(tab_data, "regions_calib_3")
```

Comment:
I think this comes from copying the code of `FastEMROCController`, as there is multiple calibrations, it'd be difficult, and anyway, if it's always the same one, it doesn't really make sense to generalise it.
So you can drop the getattr too, like that:
```
self.regions_calib_3 = tab_data.regions_calib_3
```